Tests/LibWeb/Text/input/css/FontFace.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        let face = new FontFace("Some font family", "url(some_font_family.ttf)");
```

Comment:
as I see `some_font_family.ttf` does not exists and it will fallback to default font, right? likely it would be better idea to use real font, so the test hits "success" code path.

Reply:
The code as is doesn't actually parse any fonts, nor try to load any files. the test is just about the property setters. In a real implementation (like firefox) the FaceFace has a loaded promise that is rejected, and a status of Error. There's no fallbacks in this API.